internal/pkg/bulk/opBulk.go

```diff
@@ -78,11 +81,11 @@ func (b *Bulker) writeMget(buf *Buf, index, id string) error {
 		return err
 	}
 
-	buf.WriteString(`{"_index":"`)
-	buf.WriteString(index)
-	buf.WriteString(`","_id":"`)
-	buf.WriteString(id)
-	buf.WriteString(`"},`)
+	_, _ = buf.WriteString(`{"_index":"`)
```

Comment:
We do a lot of separate writes like this to the buffer (implemented in `pkg/danger`).
The buffer implementation will never return an error.